### PR TITLE
feat: allow app secret edit

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

resolve LOG-9703, allowing app secret name editing

## Testing

- tested locally
- added integration tests

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [x] integration tests
- [ ] necessary TSDoc comments

### CODE REVIEW

**Review by wangsijie — APPROVED** [review]

**Review by simeng-li — APPROVED** [review]
core updates checked LGTM

**Review by xiaoyijun — APPROVED** [review]
LGTM

**Review by charIeszhao — APPROVED** [review]

File: packages/core/src/middleware/koa-security-headers.ts [hunk 1]
```diff
@@ -38,7 +38,13 @@ export default function koaSecurityHeaders<StateT, ContextT, ResponseBodyT>(
...
   const developmentOrigins = isProduction
     ? []
-    : ['ws:', ...['6001', '6002', '6003'].map((port) => `ws://localhost:${port}`)];
+    : [
+        'ws:',
+        ...['6001', '6002', '6003'].flatMap((port) => [
+          `ws://localhost:${port}`,
+          `http://localhost:${port}`,
+        ]),
+      ];
```
Comment (charIeszhao) [inline packages/core/src/middleware/koa-security-headers.ts:47]: Is this a relative change?

Comment (gao-sun) [reply packages/core/src/middleware/koa-security-headers.ts:47]: discussed offline, it's a by-the-way fix

**Pull request merged**